homeassistant/components/apple_tv/media_player.py

```diff
@@ -100,7 +109,7 @@ def name(self):
     @property
     def unique_id(self):
         """Return a unique ID."""
-        return self.atv.metadata.device_id
+        return f"mp_{self._identifier}"
```

Comment:
unique_id is unique per platform

Reply:
Yeah, I'm prefixing it with `mp_`, so it should be fine?

Reply:
What I meant is that you don't need to. There will hardly be another primary integration using the Same serial number

Reply:
Ah, right, now I get it! Same applies to `remote` as well then.

Reply:
Exactly. I just didn't want to repeat the comment

Reply:
Indeed

```suggestion
        return self._identifier
```